src/sentry/static/sentry/app/components/header/index.jsx

```diff
@@ -22,6 +52,13 @@ const Header = React.createClass({
     }
 
     // NOTE: this.props.orgId not guaranteed to be specified
+    let percentage = Math.round((this.getOrganization().onboardingTasks.filter(
+      (t) => { if (t.status == 'Complete') { return t; } }
```

Comment:
`Array.prototype.filter` only needs to [return true/false](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter#Parameters) (or a truthy value). This can be rewritten as:

```javascript
t => t.status === 'Complete'
```

Note that _single-line_ ES6 arrow functions have an implicit return value.